src/notificationmanager.h

```diff
@@ -115,6 +115,8 @@ public Q_SLOTS:
      * \param accountPath Notifications of this account are to be removed.
      */
     void removeNotifications(const QString &accountPath, const QList<int> &removeTypes = QList<int>());
+    void removeConversationNotifications(const CommHistory::Recipient &recipient,
```

Comment:
Minor detail but this wouldn't need to be a slot, would suffice having it in the public: section above.